### PR TITLE
Welcome screen

### PR DESCRIPTION
**Pull request by lemng32**
Implemented the UI for the welcome screen, including:
- Progress bar
- Welcome message
- Start button

### CODE REVIEW

File: frontend/components/ProgressBar/ProgressBar.jsx
Comment (praharsh05) [inline frontend/components/ProgressBar/ProgressBar.jsx:1]: Add comments to explain the code blocks

File: frontend/templates/Onboarding/OnboardPage.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import ProgressBar from '@/components/ProgressBar';
+
+import styles from './styles';
+
+const OnboardPage = () => {
+  const renderMessage = () => {
+    return (
+      <Box {...styles.welcomeMessageBoxProps}>
+        <Typography {...styles.welcomMessageProps}>
+          Welcome to <span style={{ color: '#8653FF' }}>KAI</span> 👋
```
Comment (praharsh05) [inline frontend/templates/Onboarding/OnboardPage.jsx]: move the color to styles

**Review by praharsh05 — CHANGES_REQUESTED** [review]
Awesome Work @Lamp-Nguyen, you're on the right track! Just add comments to explain the code block and I think some changes need to be made for the StepProgressChecker or StepCompletedChecker as the terminal shows some warnings when running the app.

**Comment by peifangchiang** [comment]
Great work! As @praharsh05 mentioned, there are some warnings in the terminal needed to solved.

**Review by peifangchiang — COMMENTED** [review]
Great work! As @praharsh05 mentioned, there are some warnings in the terminal needed to solved.

**Review by praharsh05 — APPROVED** [review]
Good Work @Lamp-Nguyen, let's move to finish the progress bar and take up other screens 🦾🦾

**Pull request merged**